tina_sharma/lib/tcp.js

```diff
@@ -0,0 +1,11 @@
+var net = require('net');
+var fs = require('fs');
+
+var server = net.createServer(function(socket) {
+  var d = new Date();
+  socket.pipe(fs.createWriteStream(d + '_message.txt'));
+});
```

Comment:
pretty straight forward task, but looks good.  and your code styling looks great